contracts/MMRVerification.sol

```diff
@@ -17,14 +17,15 @@ abstract contract MMRVerification {
 
     // Contract storage:
     // - current byte32 MMR root (future: array of current + previous roots as a circular queue with size N)
-    // - current public keys of validator set (optimization: merkle tree of public keys, only store root)
+    // - current public keys of validator set (optimization: merkle tree of public keys, only store root,
+    //   future: array of current + previous roots as a circular queue with size N).
 
     /**
      * @dev Updates the current MMR root, validating the new proposed MMR root by a
      *      subset of validator signatures. Called every time a new MMR root is available.
      * @param newMmrRoot the new proposed MMR root.
-     * @param signatures the subset of validator signatures on this MMR root.
-     * @param bitmap a bitmap indicating which validators the signatures belong to.
+     * @param signatures[] the subset of validator signatures on this MMR root.
```

Comment:
This should just be 

```suggestion
     * @param signatures the subset of validator signatures on this MMR root.
```

right? (The argument name, not a type.) I guess this was my bad in the last review, seems I was a bit tired when I suggested that 🙈

Reply:
Likewise on the doc comments below.

Reply:
Cleaned these up in https://github.com/Snowfork/verification-solidity/pull/1/commits/e04cbfaee55d6f1b6d981b3f2722a246c9c91a3b.